src/app/Recordings/RecordingActions.tsx

```diff
@@ -36,7 +36,7 @@ export interface RecordingActionsProps {
   index: number;
   recording: Recording;
   sourceTarget?: Observable<Target>;
-  uploadFn: () => Observable<boolean>;
+  uploadFn: () => Observable<String>;
```

Comment:
This should be `string`, not `String`:

https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html